src/views/name.njk

```diff
@@ -56,13 +56,18 @@
           <p>
             You must check the name is not the same as an existing limited
             partnership on the
-            <a href="#/">Companies House register (link opens in a new tab)</a
+            <a href="https://find-and-update.company-information.service.gov.uk/company-name-availability">Companies House register (opens in a new tab)</a
             >. If it is, we will reject your application.
           </p>
+          <p>
+            If the name includes
+            <a href="https://www.gov.uk/government/publications/incorporation-and-names/annex-a-sensitive-words-and-expressions-or-words-that-could-imply-a-connection-with-government">sensitive words or expressions that require approval (opens in a new tab)</a
+            >. you must get permission before you register.
+          </p>
           <p>
             You should also use the
```

Comment:
"https://www.gov.uk/search-for-trade mark" will give a page not found, needs to be "https://www.gov.uk/search-for-trademark"

Reply:
Updated.